### PR TITLE
feat: Selection column also support render node

### LINKED ISSUE #3505: rowSelection不支持rowSpan和colSpan
- antd 版本: 2.1.0
- 操作系统及其版本: Linux
- 浏览器及其版本: Chrome
#### 您做了什么？

我对第一列的部分行做了rowSpan，但是希望rowSelection也进行rowSpan

现在有一些数据：

``` javascript
[{
    keyword: "键盘",
    brand: "Cherry"
},{
    keyword: "键盘",
    brand: "雷柏"
},{
    keyword: "键盘",
    brand: "雷蛇"
},{
    keyword: "手表",
    brand: "Omega"
},{
    keyword: "手表",
    brand: "劳力士"
}]
```
#### 您期待的结果是：

image
#### 实际上的结果是：

无此功能
截图中的最左列的选择框是我独立实现的选择框，与keyword列同步了rowSpan属性。如果rowSelection支持rowSpan等属性的话，我们就不需要去实现交互（全选等）了。
#### 可重现的在线演示

无

### PR DESCRIPTION
**Pull request by zombieJ**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

close #3505

### 💡 Background and solution

多年前的需求……

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Table selection column support `renderCell`.       |
| 🇨🇳 Chinese |     Table 选择列支持 `renderCell`。      |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/table/demo/row-selection-custom-debug.md](https://github.com/ant-design/ant-design/blob/table-row/components/table/demo/row-selection-custom-debug.md)
[View rendered components/table/index.en-US.md](https://github.com/ant-design/ant-design/blob/table-row/components/table/index.en-US.md)
[View rendered components/table/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/table-row/components/table/index.zh-CN.md)

### CODE REVIEW

**Comment by zombieJ** [comment]
@afc163  pls help to check the naming style.

File: components/table/demo/row-selection-custom-debug.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+order: 99
+title:
+  en-US: Custom selection group
+  zh-CN: 自定义选择项组
+only: true
```
Comment (afc163) [inline components/table/demo/row-selection-custom-debug.md]: 这个得删了

**Comment by afc163** [comment]
> 多年前的需求……

无法回收的回调函数。

**Review by afc163 — APPROVED** [review]

**Comment by afc163** [comment]
> Table 选择列支持 renderCell。

从而可以****

changelog 可以写清楚帮助用户理解这个 API 能解决什么问题。

**Pull request merged**